src/components/AgreementSections/CoveredEResourcesList.js

```diff
@@ -73,7 +74,14 @@ export default class CoveredEResourcesList extends React.Component {
       const titleInstance = get(e._object, 'pti.titleInstance', {});
       return getResourceIdentifier(titleInstance, 'eissn') || getResourceIdentifier(titleInstance, 'pissn');
     },
-    platform: e => get(e._object, 'pti.platform.name', '-'),
+    platform: e => {
+      return (
+        <div>
+          <div>{e?._object?.pti?.platform?.name ?? <NoValue />}</div>
```

Comment:
Looks like if there's no platform, then you render two instances of `NoValue` and the mockup suggests only one should be rendered.